slaves/linux-cross/arm-linux-gnueabi.config

```diff
@@ -0,0 +1,573 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
+# Generated with crosstool-ng-1.22.0's `ct-ng menuconfig` command using the default configuration
+# plus the following modifications
+#
+# Path and misc options > Prefix directory = /x-tools/${CT_TARGET}
+# Target options > Target Architecture = arm
+# Target options > Architecture level = armv5t -- (*)
```

Comment:
Yes! This is the same as the `-march` flag and there [lots of possible values](https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html) we could use here. I was using `armv5t` because that's how the Ubuntu `arm-linux-gnueabi-gcc` was configured, but I think this should match the `armv6` used [here](https://github.com/rust-lang/rust/blob/5807fbbfde3ad04820f6fa0269711c81538057ec/mk/cfg/arm-unknown-linux-gnueabi.mk#L11-L12).